### PR TITLE
Fix broken TravisCI

### PR DESCRIPTION
**Pull request by Chocobo1**
Also switch builds to use [container infrastructure](https://docs.travis-ci.com/user/workers/container-based-infrastructure/).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-
-compiler: gcc
-
-env:
-    global:
-        # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
-        #   via the "travis encrypt" command using the project repo's public key
-        - secure: "BPP6SZ50idXkIPERv5Z+Q2OCvGHHEKI27icS60VnduwT3rtwZBmS2TBPAZugVdxk8AlP9txRidyGjKrliQ8j7KquqNj8RXW86Akok0/gRSWidxUsaWccCYQ00ynDQsil8hAQw3vZHiIdRVDrVYHrvSZeIPqBpfwC/i3zCUpsEoA="
-        - BUILD=autotools
+language: cpp
```
Comment (czchen) [inline .travis.yml]: Why `cpp` here? Shall it be `c` instead?

Comment (Chocobo1) [inline .travis.yml]: You are right. That was a mistake, it's fixed now.

**Comment by czchen** [comment]
Thanks for contribution.

**Comment by PeterDaveHello** [comment]
f711162865fda9a contains unrelated indent changes ...

**Pull request merged**